Realm/Tests/MigrationTests.mm

```diff
@@ -307,6 +307,25 @@ - (void)testStringPrimaryKeyNoIndexMigration {
     [RLMRealm migrateRealmAtPath:RLMTestRealmPath()];
 }
 
+- (void)testIntPrimaryKeyNoIndexMigration {
+    // make string an int
+    RLMObjectSchema *objectSchema = [RLMObjectSchema schemaForObjectClass:MigrationPrimaryKeyObject.class];
+
+    // create without search index
+    objectSchema.primaryKeyProperty.attributes = 0;
```

Comment:
```RLMPropertyAttributes``` is gone on al-swift. Once we finish up with that we are good to go. Could also just merge those changes to master if we want them out sooner but doesn't seem worth the effort yet.